### PR TITLE
[Compute] Add new API Version and field (StorageAccountType) to SharedImageGallery swagger spec

### PR DESCRIPTION
**Pull request by jerryc3**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/gallery.json [hunk 1]
```diff
@@ -0,0 +1,1480 @@
...
+        "name": {
+          "type": "string",
+          "description": "The name of the region."
+        },
+        "regionalReplicaCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The number of replicas of the Image Version to be created per region. This property is updatable."
+        },
+		"storageAccountType": {
```
Comment (vanbasten2323) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/gallery.json]: fix indentation

**Comment by jerryc3** [comment]
Hi @jhendrixMSFT, we noticed that the service team must add "WaitForARMFeedback" to the PR since we are adding a new API version. We actually don't have permissions to do that. Do you think it's necessary for this PR? If so, could you add that label?

Thanks!
Jerry

**Comment by salameer** [comment]
ARM feedback requested :)

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/gallery.json [hunk 2]
```diff
@@ -0,0 +1,1480 @@
...
+          "type": "string",
+          "description": "Specifies the storage account type to be used to store the image. This property is not updatable.",
+          "enum": [
+            "Standard_LRS",
+            "Standard_ZRS"
+          ],
+          "x-ms-enum": {
+            "name": "StorageAccountType",
+            "modelAsString": false
```
Comment (KrisBash) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/gallery.json]: Is there a strong reason for modelAsString: false?  If you want to be able to add values within an API version, I believe this must be `true`

**Review by KrisBash — CHANGES_REQUESTED** [review]
Please see comment regarding modelAsString

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/gallery.json [hunk 3]
```diff
@@ -0,0 +1,1480 @@
+{
```
[the remark this answers is not in the document]
Comment (jerryc3) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/gallery.json]: Makes sense, I'll keep that in mind for future reference

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/gallery.json (hunk 2 above)
Comment (jerryc3) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/gallery.json]: Ah I see, I've updated to true. Thanks.

**Review by KrisBash — APPROVED** [review]
LGTM

**Comment by jhendrixMSFT** [comment]
@jerryc3 can you please rebase on top of latest master?

**Comment by jerryc3** [comment]
I have just rebased, thanks!

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/5733 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/5733/commits/0a98b503386f66a95dd02a9849e75180f792c2f7

**Review by anuchandy — APPROVED** [review]

**Comment by anuchandy** [comment]
This PR introduces `StorageAccountTypes` to Gallery resource type otherwise this api-version is exactly same as the previous api-version.

**Pull request merged**